canonical/rebac-admin
# WD-10969 - refactor: make reusable table/form component

## Done

- Make a reusable component to handle forms with tables in panels.
- Add search to panels

## QA

- Go to the add group panel
- Click on each sub-form and check that you can add, remove and search.

## Details

- https://warthogs.atlassian.net/browse/WD-10969
